packages/frontend/core/src/pages/workspace/index.tsx

```diff
@@ -82,8 +83,10 @@ export const Component = (): ReactElement => {
 
   const incompatible = useLoaderData();
   return (
-    <WorkspaceLayout incompatible={!!incompatible}>
-      <Outlet />
-    </WorkspaceLayout>
+    <AffineErrorBoundary height="100vh">
```

Comment:
Why not make `100vh` as default?

Reply:
because the `AffineErrorBoundary` may be embedded in different layouts